### DISCUSSION TITLE
[Question]: What is the recommended approach for using Lambda?

### DISCUSSION THREAD

**Discussion by mikelhamer** [awspring/spring-cloud-aws#792] [Q&A]
**Type**: Question

I don't see a `spring-cloud-aws-starter-lambda` project, and am wondering, what is the recommended way to integrate Lambda that is compatible with Spring Cloud AWS 3.x?

**Comment by maciejwalkowiak** [comment]
Which integration specifically do you have in mind? Have you had a look at https://github.com/spring-cloud/spring-cloud-function?

Let me convert this issue to discussion

**Reply by mikelhamer** [reply]
My use case is to create a Lambda function as an event source listener to an SQS queue, and to be able to test the whole thing with LocalStack. What's challenging me is how to actually create the function in a test with LocalStack, seems everything requires an existing zip or jar file. Thanks for the suggestion of Spring Cloud Function, I'll check it out :)
